packages/graphiql/resources/renderExample.js

```diff
@@ -76,18 +76,18 @@ function graphQLFetcher(graphQLParams) {
   // When working locally, the example expects a GraphQL server at the path /graphql.
   // In a PR preview, it connects to the Star Wars API externally.
   // Change this to point wherever you host your GraphQL server.
-  const isDev = !window.location.hostname.match(
-    /(^|\.)netlify\.com$|(^|\.)graphql\.org$/,
-  );
-  const api = isDev ? '/graphql' : 'https://swapi.graph.cool/';
+  const isDev = window.location.hostname.match(/localhost$/);
+  const api = isDev
+    ? '/graphql'
+    : 'https://swapi-graphql.netlify.com/.netlify/functions/index';
   return fetch(api, {
     method: 'post',
     headers: {
       Accept: 'application/json',
       'Content-Type': 'application/json',
     },
     body: JSON.stringify(graphQLParams),
-    credentials: 'include',
+    credentials: 'omit',
```

Comment:
@acao Prefer `credentials: 'same-origin'` which should Just Work :tm:. Here omitting is correct, but if someone uses this example code and changes the address to their own one then it might not work with their own stack because it's not sharing the cookies. Using `'same-origin'`  by default could reduce our support burden for issues like this.